### ISSUE TITLE
[BUG] Accessibility - Web App - Web app on desktop using NVDA as Assistive technology, the user can’t keyboard through many screens and features

### ISSUE THREAD

**Issue by acovher2** [rokwire/illinois-app#5136]
**Describe the bug**
In the Web app on desktop using NVDA as Assistive technology, the user can’t keyboard through many screens and features. 

[WCAG 2.1.1 Keyboard](https://www.w3.org/WAI/WCAG21/Understanding/keyboard.html)
To Reproduce
**Steps to reproduce the behavior:**
1.	Open app, using NVDA and Chrome browser on Windows 11 
2.	Navigate to https://api.rokwire.illinois.edu/illinois/
3.	Tab through page

**Expected behavior**
User should hear the Back button label and the Title of the h1, then the user’s next tab should be the Slider, and tabs or arrows should allow interaction with slider.  

**Actual behavior**
User can hear “Dialog, dialog, dialog, Back button” label and the Title of the h1, but can’t tab to the Slider or interact with it.

**Screenshots**

https://github.com/user-attachments/assets/81768ac7-9416-4bfd-a633-f4dc03343e28

**Testing Environment & Devices (please complete the following information):
PC:**
•	OS: Windows 11
•	Device: Dell laptop
•	Browser: Chrome 135.0.7049.116

**Comment by dobromirdobrev** [comment]
Hi @acovher2 ,

There is a new version of the Illinois web app deployed on dev https://api-dev.rokwire.illinois.edu/illinois/ .

It contains the latest updates in this ticket. They are mainly in the Onboarding panels. 

Could you please check when you have a chance? And let us know if you face any issues? 
It would be very helpful if you provide some real examples with actual and expected behaviour. Thanks!

**Comment by dobromirdobrev** [comment]
Hi again @acovher2 ,

There is a new web version 7.3.40 on dev https://api-dev.rokwire.illinois.edu/illinois/ .

There are some enhancements. 

Could you please check it when you have a chance and give us some feedback what else we should handle as well? We will need some concrete examples with actual and expected behaviour. Thanks!

**Comment by acovher2** [comment]
https://github.com/user-attachments/assets/19dc2368-d824-49e0-a7a0-2e3649717826

@dobromirdobrev This is the retesting of 5045 but the video shows the expectation. There needs to be focus on elements as the user tabs through them and the user should hear with screen reader the correct names for those elements.  Currently, screen reader is getting nothing announced and the focus is missing until "Events" in the Favorites section.

**Comment by acovher2** [comment]
only thing heard by screen reader user is "blank" The user cannot tab to say yes or no to the update message. 8.0.8 version Chrome with NVDA screen reader.

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/42bc37a6-79af-4fe1-8ab9-49511d83af4a" />
